File: chatwoot/csharp/src/OSEG.ChatwootExamples/DeleteAnAccountExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class DeleteAnAccountExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "PLATFORM_APP_API_KEY");

        try
        {
            new AccountsApi(config).DeleteAnAccount(
                accountId: 0
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AccountsApi#DeleteAnAccount: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/DeleteAnAccountExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "PLATFORM_APP_API_KEY",
});

new api.AccountsApi(configuration).deleteAnAccount(
  0, // accountId
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AccountsApi#deleteAnAccount:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
